bentoctl/terraform.py

```diff
@@ -0,0 +1,63 @@
+import os
+import json
+import subprocess
+
+from bentoctl.exceptions import BentoctlException
+
+TERRAFORM_VALUES_FILE = "bentoctl.tfvars"
+TERRAFORM_INIT_FOLDER = ".terraform"
+
+
+def terraform_run(cmd: list, return_output: bool = False):
+    try:
+        if not return_output:
+            subprocess.run(["terraform", *cmd], check=False)
+        proc = subprocess.Popen(
+            ["terraform", *cmd],
+            stdout=subprocess.PIPE,
+            stderr=subprocess.PIPE,
+        )
+        stdout, stderr = proc.communicate()
+        return proc.returncode, stdout.decode("utf-8"), stderr.decode("utf-8")
+    except FileNotFoundError:
+        raise BentoctlException(
+            "terraform not available. Please make "
+            "sure terraform is installed and available your path."
+        )
+
+
+def terraform_destroy():
+    if not is_terraform_initialised():
+        raise BentoctlException('terraform is not initialised')
+    if not os.path.exists(os.path.join(os.curdir, TERRAFORM_VALUES_FILE)):
+        raise BentoctlException(
+            f"{TERRAFORM_VALUES_FILE} not found in current directory."
+        )
+    terraform_run(["apply", "-destroy", "-var-file", TERRAFORM_VALUES_FILE, '-auto-approve'])
+
```

Comment:
having -auto-approve here would not give the users a change to see exactly what changes are being made to the infra right? They should probably have a plan phase before destroy

Reply:
Yes. It won't let user to see what changed.  I am not sure user can interact with subprocess. Also I think auto approve could be an Orion in our cli.

Reply:
since we don't pipe the stdin the user can interact with it as if we called it from the terminal, so it will be a low friction way to perform all the terraform commands and yeah, we should have the auto-approve command or a more general command like '--yes' or something.